msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerMsalController.java

```diff
@@ -931,7 +921,11 @@ public void run() {
 
     private Bundle getRequestBundleForGlobalSignOut(@NonNull final IAccount account) {
         final Bundle requestBundle = new Bundle();
-        requestBundle.putString(ACCOUNT_LOGIN_HINT, account.getUsername());
+        // TODO I think a home_account_id might be better here...
+        // TODO If we really want to use a login_hint consider that you may need to dig around
+        // TODO inside of the IAccount to find it... If there is no home tenant, you'll need to inspect
+        // TODO the claims of one of the ITenantProfiles to find what you're looking for
+        requestBundle.putString(ACCOUNT_LOGIN_HINT, "");
```

Comment:
@rpdome ^^

Reply:
I'm actually removing this - in FLW scenario, there should be only 1 sign-in account in broker - if there's more, we consider it an error state and wipe everything.